test/integration/integration/tests/all-http-methods-for-wildcard.go

```diff
@@ -41,6 +41,7 @@ var ALLHTTPMethodsForWildCard = suite.IntegrationTest{
 		testCases := []http.ExpectedResponse{
 			// test path with trailing slash for GET
 			{
+				// TODO{Amila}: Fix issue with trailing slash #758
```

Comment:
What is this comment, is it intentional?

Reply:
There's an issue with this trailing slash test. Amila asked to add a comment as a TODO